common/scripts/poll-npm-package-published.mjs

```diff
@@ -0,0 +1,42 @@
+import { get } from "https";
+import process from "process";
+
+const POLL_INTERVAL_MS = 5 * 1000; // 5 seconds in millis
+const TOTAL_TIME_TO_POLL_MS = 5 * 60 * 1000; // 5 minutes in millis
+const MAX_RETRIES = TOTAL_TIME_TO_POLL_MS / POLL_INTERVAL_MS;
+
+const makeRequest = (url) => {
+  return new Promise((resolve, reject) => {
+    get(url, (response) => {
```

Comment:
Should implement a timeout here: https://stackoverflow.com/a/42357125/16098202

Reply:
Added